dotnet-client-libraries/HowToUseMapster/HowToUseMapster/Controllers/PeopleController.cs

```diff
@@ -0,0 +1,49 @@
+﻿using Microsoft.AspNetCore.Mvc;
+
+namespace HowToUseMapster.Controllers
+{
+    [Route("api/people")]
+    [ApiController]
+    public class PeopleController: ControllerBase
+    {
+		[HttpGet("new-person")]
+		public IActionResult GetNewPerson()
+		{
+		    var person = MappingFunctions.MapPersonToNewDto();
+
+		    return Ok(person);
```

Comment:
Indentation looks off, although it probably isn't. You've probably used the mix of tabs and spaces that make the code looks like this on GitHub. Use either tabs or spaces and we'll be alright.

Reply:
Yes it was looking okay locally, however, I fixed it now!